packages-exp/auth-exp/src/core/index.ts

```diff
@@ -54,6 +54,8 @@ export function signOut(auth: externs.Auth): Promise<void> {
   return auth.signOut();
 }
 
+export { initializeAuth } from './auth/initialize';
```

Comment:
What is the entry point for it? `@firebase/auth/core`?

Reply:
No entry point for it, it's only used as a building block for the other entry points

Reply:
`initializeAuth` is a public API in the original proposal, which allows people to configure things, e.g. persistence and resolvers. Is that changed?

Reply:
No. It's just the way our exports go:

`src/core/index.ts` re-exports all of the things in core
`src/index.ts` just does `export * from './core'`;
`index.ts` does `export * from './src';` as well as the browser-specific APIs